locale/config.yaml

```diff
@@ -1,72 +0,0 @@
-# Configuration for i18n workflow.
-
-locales:
-    - en  # English - Source Language
-    - ar  # Arabic
```

Comment:
Is this only used for `i18n_tools`? I'm concerned that it will break other workflows.

What do you think?

Reply:
@OmarIthawi I've noticed this pattern in many repos. 4-5 years ago, they were [stripped out of](https://github.com/openedx/edx-bulk-grades/commit/c5c914a1660cfbb8a6a7028bcd7ad691aec113fa) `edx-platform` and had this structure as a common template. Many repos including this one, don't have a transifex project for it. So, the file is not actually used. `make pull_translations` doesn't find the translation anyway

We're removing all local translations soon, therefore, we can remove unused directories like this one now. Or should we always keep them until the last phase?

Reply:
I'd err on the side of removing them. So it should be fine to do so.